lib/systems/default.nix

```diff
@@ -65,6 +66,7 @@ rec {
         # "bfd" and "gold" both come from GNU binutils. The existence of Gold
         # is why we use the more obscure "bfd" and not "binutils" for this
         # choice.
+        # Note: The gold is been default using when kernel.execFormat = elf in `development/tools/misc/binutils/default.nix`
```

Comment:
I don't think this is accurate.  Unless I'm missing something, that just controls whether gold is built at all, not whether it's used by default.

Reply:
Got it.